stdlib/asm/crypto/fri.masm

```diff
@@ -0,0 +1,873 @@
+#! Given ω, τ on stack top, this routine computes common factor `ω/ (τ - ω)`
+#! of β, as written in formula https://github.com/0xPolygonMiden/miden-vm/issues/592#issue-1481983459
```

Comment:
IMO we should keep the documentation in the code. For example, the repos were renamed/moved recently, which breaks URLs like these.

Reply:
Yes I agree, but may be not feasible always. Probably one can set up some sort of repository where all the specifications are kept and they are referred to from implementations ( i.e. code ). Which works but we still suffer from that broken link problem.